lib/widgets/menu/movies.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import 'package:movie_card/shared/movie_class.dart';
+import 'package:movie_card/shared/ui_consts.dart';
+import 'package:movie_card/shared/custom_card.dart';
+import 'package:movie_card/widgets/menu/movie_details.dart';
+
+class Movies extends StatelessWidget {
```

Comment:
For the purpose of the exercise this is not necessary you only need to have one movie